### PR TITLE
chore: Use string key instead of number for modal action key (#1537)

### PR DESCRIPTION
**Pull request by igarashitm**

### CODE REVIEW

File: packages/ui/src/components/Form/__snapshots__/CustomAutoFields.test.tsx.snap [hunk 1]
```diff
@@ -583,21 +583,6 @@ exports[`CustomAutoFields renders \`AutoFields\` for common fields 1`] = `
               value=""
             />
           </span>
-          <div
```
[the remark this answers is not in the document]
Comment (igarashitm) [reply packages/ui/src/components/Form/__snapshots__/CustomAutoFields.test.tsx.snap:586]: ugh and workflow wants to add it back... not sure what's going on...

[the remark this answers is not in the document]
Comment (igarashitm) [reply packages/ui/src/components/Form/__snapshots__/CustomAutoFields.test.tsx.snap]: reverted the snapshot change to let the workflow pass

[the remark this answers is not in the document]
Comment (lordrip) [reply packages/ui/src/components/Form/__snapshots__/CustomAutoFields.test.tsx.snap]: How do you execute the test? a single one or all of them? sometimes the IDs are generated in sequence, so if you execute a single test, the ID is 1, but if you execute alongside others, maybe it is 4 instead.

[the remark this answers is not in the document]
Comment (igarashitm) [reply packages/ui/src/components/Form/__snapshots__/CustomAutoFields.test.tsx.snap]: I'm using `yarn workspaces foreach --all --topological-dev run test` and it fails for me with upstream main, stating that this div should be removed.
```
[@kaoto/kaoto]: FAIL src/components/Form/CustomAutoFields.test.tsx
[@kaoto/kaoto]:   ● CustomAutoFields › renders `AutoFields` for common fields
[@kaoto/kaoto]: 
[@kaoto/kaoto]:     expect(received).toMatchSnapshot()
[@kaoto/kaoto]: 
[@kaoto/kaoto]:     Snapshot name: `CustomAutoFields renders \`AutoFields\` for common fields 1`
[@kaoto/kaoto]: 
[@kaoto/kaoto]:     - Snapshot  - 15
[@kaoto/kaoto]:     + Received  +  0
[@kaoto/kaoto]: 
[@kaoto/kaoto]:     @@ -500,25 +500,10 @@
[@kaoto/kaoto]:                     step="1"
[@kaoto/kaoto]:                     type="number"
[@kaoto/kaoto]:                     value=""
[@kaoto/kaoto]:                   />
[@kaoto/kaoto]:                 </span>
[@kaoto/kaoto]:     -           <div
[@kaoto/kaoto]:     -             class="pf-v5-c-form__helper-text"
[@kaoto/kaoto]:     -           >
[@kaoto/kaoto]:     -             <div
[@kaoto/kaoto]:     -               class="pf-v5-c-helper-text"
[@kaoto/kaoto]:     -             >
[@kaoto/kaoto]:     -               <div
[@kaoto/kaoto]:     -                 class="pf-v5-c-helper-text__item"
[@kaoto/kaoto]:     -               >
[@kaoto/kaoto]:     -                 <span
[@kaoto/kaoto]:     -                   class="pf-v5-c-helper-text__item-text"
[@kaoto/kaoto]:     -                 />
[@kaoto/kaoto]:     -               </div>
[@kaoto/kaoto]:     -             </div>
[@kaoto/kaoto]:     -           </div>
[@kaoto/kaoto]:               </div>
[@kaoto/kaoto]:             </div>
[@kaoto/kaoto]:             <div
[@kaoto/kaoto]:               class="pf-v5-c-expandable-section"
[@kaoto/kaoto]:             >
[@kaoto/kaoto]: 
[@kaoto/kaoto]:       20 |     });
[@kaoto/kaoto]:       21 |
[@kaoto/kaoto]:     > 22 |     expect(wrapper?.asFragment()).toMatchSnapshot();
[@kaoto/kaoto]:          |                                   ^
[@kaoto/kaoto]:       23 |   });
[@kaoto/kaoto]:       24 |
[@kaoto/kaoto]:       25 |   it('render `NoFieldFound` when there are no fields', () => {
[@kaoto/kaoto]: 
[@kaoto/kaoto]:       at Object.toMatchSnapshot (src/components/Form/CustomAutoFields.test.tsx:22:35)
```

[the remark this answers is not in the document]
Comment (igarashitm) [reply packages/ui/src/components/Form/__snapshots__/CustomAutoFields.test.tsx.snap]: But anyway this PR is already adjusted to pass GH workflows although it fails with above locally for me.

**Review by lordrip — APPROVED** [review]

**Pull request merged**